package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@merokudao/storekit-sdk",
-  "version": "1.20.0",
+  "version": "1.20.1",
```

Comment:
The version should be aligned with the API versin. If API version is `1.21` (https://github.com/merokudao/storekit/blob/80046541b5730ca07cb2017d7ff164223671b6f3/apps/docs/public/api-specs/meroku-server.yml#L4), So this should be named as `1.21.0` and not `1.20.1`